operator/controller/src/test/java/io/apicurio/registry/operator/it/EnvITTest.java

```diff
@@ -0,0 +1,177 @@
+package io.apicurio.registry.operator.it;
+
+import io.apicurio.registry.operator.api.v1.ApicurioRegistry3;
+import io.apicurio.registry.operator.resource.ResourceFactory;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.EnvVarBuilder;
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+import static io.apicurio.registry.operator.resource.ResourceFactory.APP_CONTAINER_NAME;
+import static io.apicurio.registry.operator.resource.ResourceFactory.UI_CONTAINER_NAME;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+@QuarkusTest
+public class EnvITTest extends ITBase {
+
+    private static final Logger log = LoggerFactory.getLogger(EnvITTest.class);
+
+    // spotless:off
+    private static final String[] defaultAppEnv = new String[]{
+            "QUARKUS_PROFILE",
+            "APICURIO_CONFIG_CACHE_ENABLED",
+            "QUARKUS_HTTP_ACCESS_LOG_ENABLED",
+            "QUARKUS_HTTP_CORS_ORIGINS",
+            "APICURIO_REST_DELETION_GROUP_ENABLED",
+            "APICURIO_REST_DELETION_ARTIFACT_ENABLED",
+            "APICURIO_REST_DELETION_ARTIFACTVERSION_ENABLED",
+            "APICURIO_APIS_V2_DATE_FORMAT"
+    };
+    // spotless:on
+
+    private static final String[] defaultUIEnv = new String[] { "REGISTRY_API_URL" };
+
+    @Test
+    void testEnvVars() {
```

Comment:
nitpick: I think that it will read a bit better if we split this single test in multiple, shorter once, wdyt?

Reply:
Addressing this will be very welcome, but not a blocker.